tcms/testruns/migrations/0012_test_execusion_add_fields.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 3.1.6 on 2021-02-17 04:46
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("testruns", "0011_add_planning_fields"),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name="historicaltestexecution",
+            old_name="close_date",
+            new_name="stop_date",
+        ),
+        migrations.RenameField(
+            model_name="testexecution",
+            old_name="close_date",
+            new_name="stop_date",
+        ),
+        migrations.AlterField(
+            model_name="testexecution",
+            name="stop_date",
+            field=models.DateTimeField(db_index=True, null=True),
```

Comment:
Causes `sanity / test-for-missing-migrations` to fail b/c the way you alter the fields in the DB results in them being slightly different from the models. See what the failing test does and replicate it by hand to see the difference.